frontend/src/pages/samples/SamplesPage.tsx

```diff
@@ -28,14 +38,19 @@ export default function SamplesPage() {
         };
         return {
           OR: cohortSampleFilters.concat(sampleMetadataFilters),
-        } as SampleWhere;
+          ...(_.isEqual(columnDefs, ReadOnlyCohortSampleDetailsColumns) && {
+            hasMetadataSampleMetadata_SOME: {
+              OR: sampleFilterWhereVariables(WES_SAMPLE_FILTERS),
+            },
+          }),
+        };
```

Comment:
When users click on a tab, if that tab is "View WES samples", we will include this extra "filter" with the WES criteria. This filter falls outside the `OR` clause so it will always be included in subsequent queries/searches.